Cargo.lock.msrv

```diff
@@ -104,9 +104,9 @@ checksum = "a374c89b9db55895453a74c1e38861d9deec0b01b405a82516e9d5de4820dea1"
 
 [[package]]
 name = "libc"
-version = "0.2.102"
+version = "0.2.112"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "a2a5ac8f984bfcf3a823267e5fde638acc3325f6496633a5da6bb6eb2171e103"
```

Comment:
@asomers I ran `cargo update -Zminimal-versions` and copied the generated Cargo.lock file over this one. Is that the expected workflow?